tests/Support/Models/Post.php

```diff
@@ -15,11 +16,17 @@
  * @property string|null $body
  * @property int|null $user_id
  * @property bool $flag
+ * @property Illuminate\Support\Carbon|null $published_at
+ * @property bool $is_published
```

Comment:
Nice 👍 

I did evaluate to bring Laravels ide-helper packer on board here but I think the outcome was that it doesn't support packages (or would need too much work to be made work or something).